components/SocialSharing/FacebookF.tsx

```diff
@@ -0,0 +1,7 @@
+export const FacebookF = ({ ...props}: React.SVGProps<SVGSVGElement>) => (
+  <svg {...props} xmlns="http://www.w3.org/2000/svg" viewBox="0 0 320 512" className="fill-current">
+    {/*!-- Font Awesome Free 6.4.2 by @fontawesome - https://fontawesome.com License -
+    https://fontawesome.com/license (Commercial License) Copyright 2023 Fonticons, Inc. --*/}
+    <path d="M279.14 288l14.22-92.66h-88.91v-60.13c0-25.35 12.42-50.06 52.24-50.06h40.42V6.26S260.43 0 225.36 0c-73.22 0-121.08 44.38-121.08 124.72v70.62H22.89V288h81.39v224h100.17V288z" />
+  </svg>
+);
```

Comment:
Using FontAwesome is really annoying on Nextjs
https://fontawesome.com/docs/web/use-with/react/use-with

I tried installing the React packages, and it didn't work out of the box of course.  Found out I have to do a bunch of ugly workarounds. It's simply not worth it for 5 icons so I copied the SVG on the site and convert them into JSX. Much easier this way and doesn't require weird workarounds.